### PR TITLE
Test coverage for the Drupal 8 version

### LINKED ISSUE #24: Add test coverage

### PR DESCRIPTION
**Pull request by mxr576**
Closes #24 #22

### CODE REVIEW

**Comment by mxr576** [comment]
Well, it seems it is impossible to test edge cases of the procedural code in the module file (for example because `file_exists` cannot be replaced/mocked in a test that tests procedural code in the global namespace), but at least we have a test coverage now for the most critical functionalities. Of course, excepts blocking the installation of the module if the library is missing - but that can not be tested because of the above-mentioned reasons. (Unless I actually uninstall/remove a Swagger UI lib from the test env in a test which I would prefer not to do.)

File: tests/modules/swagger_ui_formatter_test/config/optional/core.entity_view_mode.node.teaser.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+langcode: en
```
Comment (boobaa) [inline tests/modules/swagger_ui_formatter_test/config/optional/core.entity_view_mode.node.teaser.yml]: As this (testing) module depends on core's node.module, which provides this very same view mode (from its `config/install/` directory), this file is not needed here at all.

File: tests/modules/swagger_ui_formatter_test/swagger_ui_formatter_test.module [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?php
```
Comment (boobaa) [inline tests/modules/swagger_ui_formatter_test/swagger_ui_formatter_test.module]: Drupal 8 doesn't need (quasi) empty `.module` files.

File: tests/src/FunctionalJavascript/FieldFormatterTest.php [hunk 3]
```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Drupal\Tests\swagger_ui_formatter\src\FunctionalJavascript;
+
+use Drupal\Core\StreamWrapper\PublicStream;
```
Comment (boobaa) [inline tests/src/FunctionalJavascript/FieldFormatterTest.php]: Missing copyright block.

Comment (mxr576) [reply tests/src/FunctionalJavascript/FieldFormatterTest.php]: We do not have a copyright for this OS module.

**Comment by mxr576** [comment]
@balazswmann so this PR is only waiting for your review and our call before that ;)
